cmd/gocritic/main.go

```diff
@@ -62,7 +62,7 @@ func main() {
 	if cmd == nil {
 		terminate("unknown sub-command: "+sub, printSupportedSubs)
 	}
-	log.SetPrefix(sub + ": ")
+	log.SetPrefix("")
```

Comment:
I think default prefix is an empty string already, isn't it?

Reply:
I think the call to `log.SetPrefix` should be removed completely.